src/abis/ArchaeologistFacet.ts

```diff
@@ -0,0 +1,384 @@
+export const ArchaeologistFacetABI = [
```

Comment:
I'm actually kind of torn on moving the individual ABI to the Facet folder (so EmbalmerFacetABI would live in the root of EmbalmerFacet folder). This would seem better for this app, but that might be slightly confusing when we need to update.

Reply:
I'll delete the other ABIs.